src/phpcs/Production/Sniffs/Commenting/FunctionCommentSniff.php

```diff
@@ -13,7 +13,7 @@
  * @codingStandardsIgnoreFile duplicated vendor code. Removed requiring parameter, throws comment
  * @SuppressWarnings(PHPMD) duplicated vendor code. Removed requiring parameter, throws comment
```

Comment:
Can this annotation be removed? You changed the class inheritance, so I assume this might no longer be required.

Reply:
These lines disable that any coding style or PMD checks are being done on this file.
That is still necessary, because the code would result in plenty failures,  the following are from phpmd:

```
[…]/FunctionCommentSniff.php:59	The method process() has a Cyclomatic Complexity of 12. The configured cyclomatic complexity threshold is 10.
[…]/FunctionCommentSniff.php:59	The method process() has an NPath complexity of 240. The configured NPath complexity threshold is 200.
[…]/FunctionCommentSniff.php:59	The method process() has 58 lines of code. Current threshold is set to 39. Avoid really long methods.
[…]/FunctionCommentSniff.php:129	The method processReturn() has a Cyclomatic Complexity of 38. The configured cyclomatic complexity threshold is 10.
[…]/FunctionCommentSniff.php:129	The method processReturn() has an NPath complexity of 15776. The configured NPath complexity threshold is 200.
[…]/FunctionCommentSniff.php:129	The method processReturn() has 172 lines of code. Current threshold is set to 39. Avoid really long methods.
[…]/FunctionCommentSniff.php:169	Avoid unused local variables such as '$split'.
[…]/FunctionCommentSniff.php:180	Avoid unused local variables such as '$i'.
[…]/FunctionCommentSniff.php:308	The method processParams() has a Cyclomatic Complexity of 55. The configured cyclomatic complexity threshold is 10.
[…]/FunctionCommentSniff.php:308	The method processParams() has an NPath complexity of 1394669556. The configured NPath complexity threshold is 200.
[…]/FunctionCommentSniff.php:308	The method processParams() has 326 lines of code. Current threshold is set to 39. Avoid really long methods.
[…]/FunctionCommentSniff.php:535	Avoid unused local variables such as '$line'.
[…]/FunctionCommentSniff.php:645	The method processThrows() has a Cyclomatic Complexity of 10. The configured cyclomatic complexity threshold is 10.
[…]/FunctionCommentSniff.php:645	The method processThrows() has 51 lines of code. Current threshold is set to 39. Avoid really long methods.
[…]/FunctionCommentSniff.php:649	Avoid unused local variables such as '$throws'.
[…]/FunctionCommentSniff.php:707	The method suggestType() has a Cyclomatic Complexity of 18. The configured cyclomatic complexity threshold is 10.
[…]/FunctionCommentSniff.php:707	The method suggestType() has an NPath complexity of 200. The configured NPath complexity threshold is 200.
[…]/FunctionCommentSniff.php:707	The method suggestType() has 67 lines of code. Current threshold is set to 39. Avoid really long methods.
```